node/src/transaction_pool/candidate/transaction_pool_candidate_actions.rs

```diff
@@ -37,12 +38,18 @@ pub enum TransactionPoolCandidateAction {
         peer_id: PeerId,
         transaction: TransactionWithHash,
     },
+    TransactionsReceived {
```

Comment:
@0xMimir I think this should have some mention of libp2p in the name, or at least a comment making it explicit that this is libp2p specific